scripts/build-in-docker.sh

```diff
@@ -4,11 +4,14 @@ set -eux
 DOCKER_IMAGE="getsentry/rust-musl-cross:${DOCKER_TAG}"
 BUILD_DIR="/work"
 
+# TODO: Remove OPENSSL_NO_VENDOR once openssl-src >111.11.0 is released.
```

Comment:
openssl-src 111.14.0 has been released. If it still doesn't work, let's update or remove the comment.

We control the openssl version in our fork of the rust-musl-cross container. Once we're able to vendor it on all architectures, we can get rid of that change in the fork.